web/views/recurring_jobs.erb

```diff
@@ -1,50 +1,74 @@
-        <th><%= t('description') %></th>
-        <th><%= t('interval') %></th>
-        <th><%= t('class') %></th>
-        <th><%= t('queue') %></th>
-        <th><%= t('arguments') %></th>
-        <th><%= t('last_time') %></th>
-        <th><%= t('next_time') %></th>
-        <th></th>
-      </tr>
-    </thead>
+<style type="text/css">
+  .recurring-jobs { border-top-left-radius: 4px; border-top-right-radius: 4px; }
+  .recurring-jobs .title { margin-bottom: 5px; }
+  .recurring-jobs .title .name { font-weight: bold;}
+  .recurring-jobs .info,
+  .recurring-jobs .description { margin-bottom: 5px; }
+  .recurring-jobs .actions { margin-bottom: 5px; }
+  .recurring-jobs .status,
+  .recurring-jobs .description { font-size: 12px; }
+  .recurring-jobs .enqueue { margin-bottom: 0.5rem }
 
-    <tbody>
-      <% @presented_jobs.each do |job| %>
-        <tr>
-          <td><%= job.name %></td>
-          <td><%= job['description'] %></td>
-          <td><%= job.interval %></td>
-          <td><%= job['class'] %></td>
-          <td>
-            <a href="<%= root_path %>queues/<%= job.queue %>"><%= job.queue %></a>
-          </td>
-          <td><%= job['args'] %></td>
-          <td><%= job.last_time %></td>
-          <td>
-            <span style="<%= 'text-decoration:line-through' unless job.enabled? %>">
-              <%= job.next_time || t('no_next_time') %>
-            </span>
-          </td>
-          <td class="text-center">
-            <form action="<%= root_path %>recurring-jobs/<%= ERB::Util.url_encode(job.name) %>/enqueue" method="post">
+  .list-group-item {
+    background-color: #222;
+    color: white;
+    border: 1px solid #555;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    .list-group-item {
+      background-color: #222;
+      color: white;
+      border: 1px solid #555;
+    }
+  }
+</style>
+
+<div class="recurring-jobs">
+  <ul class="list-group">
+    <% @presented_jobs.each do |job| %>
+      <li class="list-group-item">
+        <div class="title">
+          <div class="row">
+            <div class="col-xs-6">
+              <span class="name"><%= job.name %></span>
+            </div>
+            <div class="col-xs-6 text-right">
+              <a href="<%= root_path %>queues/<%= job.queue %>"><%= job.queue %></a>
+            </div>
+          </div>
+        </div>
+        <div class="description"><%= job['description'] %></div>
+        <div class="info">
+          <div class="row">
+            <div class="col-md-4 class"><%= job['class'] %></div>
+            <div class="col-md-4 interval text-left"><%= t('interval') %>: <%= job.interval %></div>
+            <div class="col-md-4 args"><%= t('arguments') %>: <%= job['args'] %></div>
+          </div>
+        </div>
+        <div class="status row">
+          <div class="col-md-4 actions">
+            <form action="<%= root_path %>recurring-jobs/<%= ERB::Util.url_encode(job.name) %>/enqueue" method="post" class="enqueue">
```

Comment:
is there not already a helper for this route that you can use here, where you pass the job's name via param?

Reply:
Not that I'm aware, at least sidekiq doesn't uses something similar https://github.com/mperham/sidekiq/blob/master/web/views/scheduled.erb#L15